javascript/examples/custom-observability/test-no-auto-init.ts

```
@@ -3,24 +3,55 @@
+  const provider = trace.getTracerProvider() as { getDelegate?: () => object };
+  return (provider.getDelegate ? provider.getDelegate() : provider).constructor
+    .name;
+};
+
 // Check the provider BEFORE importing scenario
-const providerBefore = trace.getTracerProvider();
-const providerNameBefore = providerBefore.constructor.name;
+const delegateBefore = delegateName();
 
 // Dynamically import scenario to test the side-effect
 const scenario = await import("@langwatch/scenario");
```

ESLint

[error] Unable to resolve path to module '`@langwatch/scenario`'.

(import/no-unresolved)